requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+# Core
+Cython>=3.0a4
+typing-extensions
```

Comment:
There requirements get installed by `python setup.py develop`. Unfortunately I don't see how to specify the test ones (`python setup.py test` is deprecate). The dependencies are specified in `tox.ini`: I'll write something in the comments: please tell me if it makes sense.